src/Blog/Entity/PostTag.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Blog\Entity;
+
+use Cycle\Annotated\Annotation\Column;
+use Cycle\Annotated\Annotation\Entity;
+
+/**
+ * @Entity
+ */
+class PostTag
+{
+    /**
+     * @Column(type="primary")
+     */
+    private ?int $id = null;
```

Comment:
Doesn't seem to be used anywhere.

Reply:
Do we need a surrogate primary key for post_tag? Can't Cycle use compound post_id, tag_id key?

Reply:
> compound post_id, tag_id key
This is not yet supported.